### DISCUSSION TITLE
Wordpress 6.2.1 upgrade - [tsml_react] no longer displays meetings

### DISCUSSION THREAD

**Discussion by conrac01** [code4recovery/12-step-meeting-list#1134] [Q&A] [answered]
After upgrading to 6.2.1 the meetings disappeared.   I tried changing UI, deactivating and reactivating plugin.  I changed timezone in WP settings to UTC-4 and then back to New York.  Also, when I press "Where's my Info"?  the /meetings page can't be found (404).

I have a backup, but I'd prefer to learn how to fix it as the same thing happened before and one of the above fixed it.

site is https://district2.nenyaa.org

Many Thanks - Ed

**Comment by joshreisner** [comment]
hm, maybe go to wordpress settings > permalinks > save and see if that fixes the 404 at `/meetings`

not sure why the shortcode would not be parsed at https://district2.nenyaa.org/district-2-meetings -- maybe someone else has an idea?

**Reply by conrac01** [reply]
Using 6.2.1 with 2023 Theme - blocks only.  I change the page template from meetings to my front page template and the problem corrected.

I was getting the 404 from within the dashboard/Meetings/Import & Export in the section "Where's my info"?

After changing the page template the 404 disappeared.

Many Thanks.....

**Comment by anchovie91471** [comment, marked as answer]
Hi Ed!

How is the shortcode inerted into the page? Is it block editor or classic editor or template?

**Reply by conrac01** [reply]
Using 6.2.1 with 2023 Theme - blocks only. I change the page template from meetings to my front page template and the problem corrected.

I was getting the 404 from within the dashboard/Meetings/Import & Export in the section "Where's my info"?

After changing the page template the 404 disappeared.

Your editor suggestion put me on the right track..... Thanks

**Reply by anchovie91471** [reply]
Glad it helped!! So you are using a shortcode block to enter the [tsml_react]?

**Reply by conrac01** [reply]
Yes, I'm using the [tsml_react] shortcode.

**Reply by anchovie91471** [reply]
Sounds like you got it fixed so all is good. I was wondering mostly what type of block you used.
